backend/de.metas.business/src/main/java/de/metas/invoice/service/impl/AbstractInvoiceBL.java

```diff
@@ -161,18 +161,23 @@
 	public final I_C_Invoice creditInvoice(@NonNull final I_C_Invoice invoice, final InvoiceCreditContext creditCtx)
 	{
 		Check.errorIf(isCreditMemo(invoice), "Param 'invoice'={} may not be a credit memo");
-		Check.errorIf(invoice.isPaid(), "Param 'invoice'={} may not yet be paid");
-
 		Check.assume(invoice.getGrandTotal().signum() != 0, "GrandTotal!=0 for {}", invoice);
 
-		final Properties ctx = InterfaceWrapperHelper.getCtx(invoice);
-		//
-		// 'openAmt is the amount that shall end up in the credit memo's GrandTotal
-		final BigDecimal openAmt = Services.get(IAllocationDAO.class).retrieveOpenAmt(invoice,
-				false); // creditMemoAdjusted = false
+		if (creditCtx.isCreditedInvoiceReinvoicable())
```

Comment:
are we sure this should be `isCreditedInvoiceReinvoicable()` and not `referenceInvoice()`